tokio-util/tests/task_join_map.rs

```diff
@@ -343,3 +345,34 @@ async fn duplicate_keys2() {
 
     assert!(map.join_next().await.is_none());
 }
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
There are downstream users of Tokio who compile with panic=abort in cases other than wasm.
```suggestion
#[cfg_attr(not(panic = "unwind"), ignore)]
```

Reply:
Also I believe this fixes the unused import warnings since `#[cfg(ignore)]` just means the test doesn't run - it's still compiled and can use imports.